gravitee-apim-portal-webui-next/src/entities/api/api-information.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface ApiInformation {
+  /**
+   * Name of api information
+   */
+  name?: string;
+  /**
+   * Value of api information
+   */
+  value?: string;
```

Comment:
I think these don't need to be conditional. You can remove the `?`

Reply:
But they are conditional in old portal. Alright, we can optimize it.

Reply:
Done